### ISSUE TITLE
Unable to update Formie from 1.2.27 => 1.2.28

### ISSUE THREAD

**Issue by howells** [verbb/formie#223]
**Description**

When I try to update 1.2.27 => 1.2.28, I get the following issue:

```
<warning>Package "craftcms/vue-asset" listed for update is not installed. Ignoring.</warning>
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for verbb/formie 1.2.28 -> satisfiable by verbb/formie[1.2.28].
    - Conclusion: remove fzaninotto/faker v1.9.1
    - Conclusion: don't install fzaninotto/faker v1.9.1
    - verbb/formie 1.2.28 requires fakerphp/faker ^1.9 -> satisfiable by fakerphp/faker[v1.11.0, v1.10.1, v1.10.0, v1.9.1].
    - fakerphp/faker v1.11.0 conflicts with fzaninotto/faker[v1.9.1].
    - fakerphp/faker v1.10.1 conflicts with fzaninotto/faker[v1.9.1].
    - fakerphp/faker v1.10.0 conflicts with fzaninotto/faker[v1.9.1].
    - don't install fakerphp/faker v1.9.1|don't install fzaninotto/faker v1.9.1
    - Installation request for fzaninotto/faker (locked at v1.9.1) -> satisfiable by fzaninotto/faker[v1.9.1].

<warning>Running update with --no-dev does not mean require-dev is ignored, it just means the packages will not be installed. If dev requirements are blocking the update you have to resolve those problems.</warning>
```

I have removed the vendor folder, deleted the composer cache, removed the lock file and updated from scratch, etc. but these steps haven't worked.

**Additional info**
- Plugin version: 1.2.27
- Craft version: 3.5.15.1

**Comment by engram-design** [comment]
After talking with the Craft team (cc @brandonkelly), it looks like there’s an issue with Composer in this situation. I will assume you’re using either the control panel to update the plugin, or using `./craft update all` - correct?

I would recommend to change the following in your `composer.json` file:

```
"verbb/formie": "1.2.27",
```

to 
```
"verbb/formie": "1.2.28",
```

and run `composer update`. This at least worked for me. I’m continuing to look at ways to better handle this.

This stems from the fact that in `1.2.28` we swapped `fzaninotto/faker` to `fakerphp/faker` due to the former being abandoned. I can't figure out why it's trying to install `fzaninotto/faker` if it has been removed. There's decidedly different behaviour when running `./craft update all` and `composer update`.

**Comment by engram-design** [comment]
Refer also to https://github.com/verbb/formie/issues/218#issuecomment-731090779

**Comment by howells** [comment]
@engram-design yeah that worked, thanks!

**Comment by engram-design** [comment]
I can confirm this'll be fixed in Craft 3.6, thanks to the update to Composer 2.
